src/components/views/emojipicker/ReactionPicker.tsx

```diff
@@ -119,9 +121,18 @@ class ReactionPicker extends React.Component<IProps, IState> {
         }
     };
 
+    private isEmojiDisabled = (unicode: string): boolean => {
+        if (!this.getReactions()[unicode]) return false;
+        if (this.props.mxEvent.isRedacted()) return false;
```

Comment:
this seems backwards, surely this should be return true?

Reply:
Do you mean the emoji button should be disabled if we don't have a reaction of its kind yet?

Reply:
This looks to be saying, if the target event isRedacted then isEmojiDisabled=false, surely we shouldn't allow users to react to already redacted events

Reply:
Oh, you mean the other line :D Do we ever show the `ReactionRow` for deleted events

Reply:
If we don't then why is this line here?

Reply:
Because I was confused and actually meant to write something else :sweat_smile: - the line is now gone